vault/changelog.release.2021-12-14.md

```diff
@@ -0,0 +1,85 @@
+
+### Highlights
+
+- feat(commands): find broken links ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#findbrokenlinks]])
+- feat(notes): Note Trait System Prototype (Phase 1) ([[docs|dendron://dendron.dendron-site/dendron.topic.traits.quickstart]])
+- feat(navigation): implement goto definition for non-note files
+- feat(navigation): `Goto Note` can open links to non-note files
+- enhance(navigation): allow `Goto Note` to work inside code blocks
+- feat(refactor): add rename provider
+
+### Everything Else
+
+- feat(pods): early version of Orbit import pod ([[docs|dendron.topic.pod.builtin.orbit]])
+- enhance(markdown): add `depth` metadata to header anchors
+- enahnce(workspace): simplify `InitializeWorkspace` command
+- enhance(publish): better layout for Table of Contents / TOC
+- enhance(pods): small tweaks to pod v2 UI
+- enhance(publish): Fallback to default SEO image if no image is set for published pages.
+- fix(workspace): tutorial initializer with existing workspace in default paths
+- fix(refactor): revert match text default value to active note name
+- fix(extension): note traits not working after webpack
+- fix(schema): use patterns when ids are auto generated and there is no manually set title for a schema
+- fix(views): double open link from preview
+
+### Community
+
+#### General Updates
+
+Dendron was recently featured in the _Tools for Thought_ series by Ness Labs. Kevin Lin gave the rundown on Dendron features, suggestions for new users, and what the future looks like.
+
+- [Ness Labs: Redefining knowledge management with Kevin Lin, founder of Dendron](https://link.dendron.so/6cqa)
+
+Dendron is soon to be featured in the next VS Code livestream!
+
+> In our talk, we'll go over the basics of how Dendron works and do a walkthrough of the core features that let individuals and teams manage tens of thousands of notes inside of VS Code.
+
+- [VSCode Livestreams](https://code.visualstudio.com/livestream): Note taking inside of VS Code (with Dendron)
+  - **Thursday, December 16, 8:00 AM PST**
+
+#### Dendron Reading Series
+
+This week's entry in the [[Dendron Reading Series|community.reading-series]]. 
+
+![[11 - Cal Newport's Study Hacks Blog: In Defense of Thinking|dendron://dendron.dendron-site/community.reading-series.journal.2021.12.14]]
+
+#### Office Hours
+
+- We will be hosting Office Hours next week! Event details [here](https://link.dendron.so/6cqo).
+- Notes, and the recording, of the previous Office Hours can be found [[here|dendron://dendron.dendron-site/community.office-hours.2021.12.08]]
+
+#### Thank You's
+
+A big **thanks** to the following gardeners that brought up issues, contributions, and fixes to this release! You can see an overview of all roles [[here|dendron://dendron.dendron-site/community.roles]]
+
+- [Tika](https://github.com/SR--) `@Tika#9526`
```

Comment:
can we add community doc contributions? specifically merge https://github.com/dendronhq/dendron-site/pull/318/files and acknowledge?